packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -113,6 +133,25 @@ export default class Menu extends Component {
     return signOut()
   }
 
+  _subscribeMissingPatches = () => {
+    const unsubs = map(this.props.hosts, host =>
+      subscribeHostMissingPatches(host, patches => {
+        this.setState({
+          missingPatches: {
```

Comment:
The number of missing patches per host isn't displayed, so what we need to know is if a patch is missing on a host. There is my suggestion:
```js
subscribeHostMissingPatches(host, patches => {
  if (patches.length > 0) {
    this.setState({
      isSomePatchesMissing: true,
     })
   }
})
```